packages/kit/src/core/create_app/index.js

```diff
@@ -21,11 +25,11 @@ export function write_if_changed(file, code) {
 /** @typedef {import('types/internal').ManifestData} ManifestData */
 
 /**
- * @param {{
- *   manifest_data: ManifestData;
- *   output: string;
- *   cwd: string;
- * }} options
+ * Updates or creates two files - manifest.js and root.svelte - in the specified output directory.
+ * @param {Object} options
+ * @param {ManifestData} options.manifest_data - details the apps assets, user defined components, routes, and __layout and error components
```

Comment:
I worry this description has the potential to get outdated if we change what is contained within the `ManifestData`. I'd rather ensure that the `ManifestData` type is well documented and make this description more minimal

Reply:
Agreed. Probably best to just refer to the type declaration for any objects which are typed.